.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This file should be probably in a different PR as it’s unrelated to this one.

Reply:
Fair enough.
I'll move linting to another PR.